package.json

```diff
@@ -55,23 +57,30 @@
     "@babel/plugin-transform-parameters": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.6.0",
     "@babel/register": "^7.0.0",
     "@emotion/babel-preset-css-prop": "10.0.9",
     "@storybook/addon-a11y": "^5.1.9",
     "@storybook/addon-actions": "^5.1.9",
     "@storybook/addon-centered": "^5.1.9",
+    "@storybook/addon-info": "^5.2.5",
```

Comment:
Was this added because it was a missing `peerDependency` of something?

Reply:
done, I dont remeber adding this I vaguely remeber using withInfo but it doesn seem to exist anymore